app/controllers/historic_appointments_controller.rb

```diff
@@ -0,0 +1,44 @@
+class HistoricAppointmentsController < ApplicationController
+  def show
+    @past_prime_minister = PastPrimeMinister.find!(request.path)
+    setup_content_item_and_navigation_helpers(@past_prime_minister)
+  end
+
+  def index
+    if request.path.include?("past-prime-ministers")
+      @index_page = PastPrimeMinistersIndex.find!(request.path)
+      @presenter = PastPrimeMinistersIndexPresenter.new(@index_page.all_prime_ministers)
+    else
+      @index_page = HistoricAppointmentsIndex.find!(request.path)
+      @presenter = HistoricAppointmentsIndexPresenter.new(@index_page)
+    end
+    setup_content_item_and_navigation_helpers(@index_page)
+  end
+
+  def show_past_foreign_secretary
+    setup_content_item_and_navigation_helpers(HistoricAppointmentsIndex.find!(request.path.split("/")[0...-1].join("/")))
```

Comment:
I know this is originally imported code, but I wonder if we should improve it whilst you're in the area! We could just pass in`request.path` rather than `request.path.split("/")[0...-1].join("/")` and add a bit more logic to the HistoricAppointmentsIndex find method to figure out which file to read. Eg:

```
def self.find!(base_path)
    directory =
      base_path.include?("past-foreign-secretaries") ? "past_foreign_secretaries" : "past_chancellors"
    yaml_content = YAML.load_file(Rails.root.join("config/#{directory}/content_item.yml"))
    new(ContentItem.new(yaml_content))
  end
```
Not a blocker, just a style thing.